### ISSUE TITLE
Node run failed: error: operation not permitted

### ISSUE THREAD

**Issue by yuweimian-shy** [langgenius/dify-sandbox#53]
I followed the steps in FAQ.md and got the following error at step2, anyone else having the same problem?
![微信图片_20240830103620](https://github.com/user-attachments/assets/9bfb7408-93f8-4ad0-9af0-7687ee6ad565)

**Comment by yuweimian-shy** [comment]
When I run this test file alone, I get the following error
![微信图片_20240830110409](https://github.com/user-attachments/assets/96a154fe-7557-4f84-8df2-003e7099196d)

**Comment by Hanfee** [comment]
Please show your test.py , 

Before test your python script, please read the README.md

**Comment by yuweimian-shy** [comment]
i just add “import numpy” before main function following the FAQ.md, The rest is unchanged.
Is that right? or i need to change my code?

> Please show your test.py ,
> 
> Before test your python script, please read the README.md

**Comment by yuweimian-shy** [comment]
> Please show your test.py ,
> 
> Before test your python script, please read the README.md

i just add “import numpy” before main function following the FAQ.md, The rest is unchanged.
Is that right? or i need to change my code?
![企业微信截图_17250069512128](https://github.com/user-attachments/assets/5614c688-408c-4df4-85da-90a87be81cba)

**Comment by Hanfee** [comment]
Is the user sandbox present on your platform?

uid=65537(sandbox) gid=59846(sandbox) group=59846(sandbox) — This system call execution is bound to the sandbox user.

**Comment by athmey** [comment]
> Is the user sandbox present on your platform?
> 
> uid=65537(sandbox) gid=59846(sandbox) group=59846(sandbox) — This system call execution is bound to the sandbox user.

yes, currently the sandbox is on the platform.（目前正在服务器上配置sandbox的环境）
OS:Ubuntu20.04
CPU arch: amd64

Following the instructions:
1.Clone the repository using git clone https://github.com/langgenius/dify-sandbox and navigate to the project directory.
2.Run ./install.sh to install the necessary dependencies.
3.Run ./build/build_[amd64|arm64].sh to build the sandbox binary.
4.Run ./main to start the server.

while doing the "./main", the output is like this:
![res](https://github.com/user-attachments/assets/337639ea-393e-4d52-8339-ec60577e9b62)

**Comment by Hanfee** [comment]
@athmey Please sure pip3 command is exists ？If not, please install it first

**Comment by athmey** [comment]
> @athmey Please sure pip3 command is exists ？If not, please install it first

Thank you so much

**Comment by wwh-coder** [comment]
@Hanfee  Why is the result of my operation like this?

Failed to run your python code, exit status 2
failed with exit status 2


uid=65537(sandbox) gid=1001(sandbox) groups=1001(sandbox)

I just add “import pandas” before main function following the FAQ.md
lib.DifySeccomp(65537, 1001, 1001)
import pandas
# declare main function here
def main() -> dict:
    return {"message": [1, 2, 3]}

![image](https://github.com/user-attachments/assets/22d2d8e9-a2a6-4bcd-be95-cc0bee28caaa)

I don't know where I went wrong, could you please clarify? Thank you very much
